cms/djangoapps/contentstore/rest_api/v1/views.py

```diff
@@ -110,6 +113,29 @@ def post(self, request, course_id):
         serializer = ProctoredExamSettingsSerializer if request.user.is_staff else LimitedProctoredExamSettingsSerializer
         exam_config = serializer(data=request.data.get('proctored_exam_settings', {}))
         valid_request = exam_config.is_valid()
+
+        # uncomment if extra validation is needed to ensure that zendesk tickets are being used correctly
+        # exam_config.validate(request.data.get('proctored_exam_settings', {}))
+
+        # log if zendesk ticket field is not appropriate for proctoring provider
+        proctoring_provider = exam_config.data.get('proctoring_provider')
+        create_zendesk_tickets = exam_config.data.get('create_zendesk_tickets', None)
+
+        if create_zendesk_tickets is not None:
+            if (
+                (proctoring_provider == 'proctortrack' and create_zendesk_tickets)
+                or (proctoring_provider == 'software_secure' and not create_zendesk_tickets)
+            ):
+                LOGGER.info(
```

Comment:
It's weird to me you have the same logic here as you do in the serializer. Why not just log in the serializer for now, and in the serializer, don't raise errors. Then in this code, you can just call the serializer validate function. Why do we want to duplicate the code this way?